src/wp-includes/rest-api/endpoints/class-wp-rest-blocks-controller.php

```diff
@@ -56,6 +57,10 @@ public function filter_response_by_context( $data, $context ) {
 		unset( $data['title']['rendered'] );
 		unset( $data['content']['rendered'] );
 
+		// Add the core wp_sync_status meta as top level property to the response.
+		$wp_sync_status         = $data['meta']['wp_sync_status'];
+		$data['wp_sync_status'] = $wp_sync_status;
```

Comment:
```suggestion
		$data['wp_sync_status'] = $data['meta']['wp_sync_status'];
```
Don't think `$wp_sync_status` needed here. WDYT?

Reply:
True, not sure why iI added that - removed.